src/Form/SsofactRegisterForm.php

```diff
@@ -129,7 +129,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['privacy'] = [
       '#type' => 'checkbox',
       '#required' => TRUE,
-      '#title' => $this->t('I accept terms and conditions'),
+      '#title' => $this->t('Please note the general <a href=":url_data">data protection regulations</a> and the <a href=":url_info">information obligation</a>.', [
+        ':url_data' => '/node/2',
+        ':url_info' => 'https://www.stimme-medien.de/metanavigation/informationspflichten/',
```

Comment:
Hardcoding these is pretty bad… but I do not see a quick solution for this right now, so we need to come back to this later.